### PR TITLE
updated tags and bumped to v1.0

### LINKED ISSUE #5514: Remote: Pre-launch: style.css tags
We need to review the tags and see if we are missing any https://make.wordpress.org/themes/handbook/review/required/theme-tags/

### PR DESCRIPTION
**Pull request by MaggieCabrera**
#### Changes proposed in this Pull Request:

This PR bumps the version to 1.0.0 and updates the tags on the style.css file to fit the theme

#### Related issue(s):

Closes https://github.com/Automattic/themes/issues/5514

### CODE REVIEW

**Comment by MaggieCabrera** [comment]
Let's merge/deploy this one last as we are ready to submit to .org

File: remote/style.css [hunk 1]
```diff
@@ -7,11 +7,11 @@ Description: Remote is a dark, minimal block theme ideal for bloggers. Its defau
 Requires at least: 5.8
 Tested up to: 5.9
 Requires PHP: 5.7
-Version: 0.0.13
+Version: 1.0.0
 License: GNU General Public License v2 or later
 License URI: https://raw.githubusercontent.com/Automattic/themes/trunk/LICENSE
 Text Domain: archeo
```
Comment (mikachan) [inline remote/style.css]: It looks like this (Text Domain) and the Theme URI just need updating.

Comment (MaggieCabrera) [reply remote/style.css]: good catch!

**Review by mikachan — APPROVED** [review]
Looking good!

**Pull request merged**